homeassistant/components/sensibo/entity.py

```diff
@@ -75,3 +75,33 @@ async def async_send_api_call(
                 params["assumed_state"],
             )
         return result
+
+
+class SensiboMotionBaseEntity(CoordinatorEntity):
+    """Representation of a Sensibo numbers."""
+
+    coordinator: SensiboDataUpdateCoordinator
+
+    def __init__(
+        self,
+        coordinator: SensiboDataUpdateCoordinator,
+        device_id: str,
+        sensor_id: str,
+        sensor_data: MotionSensor,
+        name: str | None,
+    ) -> None:
+        """Initiate Sensibo Number."""
+        super().__init__(coordinator)
+        self._device_id = device_id
+        self._sensor_id = sensor_id
+        self._client = coordinator.client
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, sensor_id)},
```

Comment:
Is `sensor_id` globally unique?

Reply:
I believe it should be pretty unique. An example:
```json
{"id" : "ms_b29diThZ"}
```

Reply:
🤞 lets hope 👍